gslib/commands/cp.py

```diff
@@ -81,194 +81,188 @@
 
 _DESCRIPTION_TEXT = """
 <B>DESCRIPTION</B>
-  The gsutil cp command allows you to copy data between your local file
-  system and the cloud, copy data within the cloud, and copy data between
+  The ``gsutil cp`` command allows you to copy data between your local file
+  system and the cloud, within the cloud, and between
   cloud storage providers. For example, to upload all text files from the
-  local directory to a bucket you could do:
+  local directory to a bucket, you can run:
 
     gsutil cp *.txt gs://my-bucket
 
-  Similarly, you can download text files from a bucket by doing:
+  You can also download text files from a bucket:
 
     gsutil cp gs://my-bucket/*.txt .
 
-  If you want to copy an entire directory tree you need to use the -r option.
-  For example, to upload the directory tree "dir":
+  Use the ``-r`` option to copy an entire directory tree.
+  For example, to upload the directory tree ``dir``:
 
     gsutil cp -r dir gs://my-bucket
 
-  If you have a large number of files to transfer you might want to use the
-  top-level gsutil -m option (see "gsutil help options"), to perform a
-  parallel (multi-threaded/multi-processing) copy:
```

Comment:
I recommend keeping this - it was added because of b/75005961, and I've seen other comments in the past from people confused about -m being top-level and not command-specific

Reply:
also, this renders as a link in our documentation, in case you weren't aware of that behavior